src/app/(dashboard)/Sidebar.tsx

```diff
@@ -55,31 +60,41 @@ function ListItem({
     selected_form_id: string
 }) {
     return (
-        <Link href={'/builder/' + form_id}>
-            <li
-                className={cn(
-                    'group cursor-pointer hover:border-primary hover:text-primary mt-3 py-2 px-3 flex gap-2 group shadow-md border rounded-xl',
-                    selected_form_id === form_id &&
-                        'border-primary text-primary font-bold'
-                )}
-            >
-                <ListIcon />
-                <span className="text-sm truncate">{form_title}</span>
-            </li>
-        </Link>
+        <HoverCard openDelay={100} closeDelay={100}>
+            <HoverCardTrigger asChild>
+                <Link href={'/builder/' + form_id}>
+                    <li
+                        className={cn(
+                            'group cursor-pointer hover:border-primary hover:text-primary mt-3 py-2 px-3 flex gap-3 shadow-md border rounded-xl md:rounded-full md:items-center md:text-center md:p-2',
+                            selected_form_id === form_id &&
+                                'border-primary text-primary font-bold'
+                        )}
+                    >
+                        <ListIcon />
+                        <span className="text-sm truncate hidden md:block">
+                            {form_title}
+                        </span>
+                    </li>
+                </Link>
+            </HoverCardTrigger>
+            <HoverCardContent side="right" className="md:hidden">
+                {form_title}
+            </HoverCardContent>
+        </HoverCard>
     )
 }
 
-export default function Sidebar({ className }: { className?: String }) {
+export default function Sidebar({ className }: { className?: string }) {
     const params = useParams()
 
     return (
         <nav className={cn('m-5 p-4 rounded-xl flex flex-col', className)}>
             <div className="flex justify-between">
                 <h1 className="font-bold text-sm ml-2 text-gray-600">
-                    My Forms
+                    <span className="block md:inline">My</span>
+                    <span className="hidden md:inline">&nbsp;</span>
+                    <span className="block md:inline">Forms</span>
                 </h1>
-                <button className="w-6 h-6 rounded-full bg-primary shadow-md border"></button>
```

Comment:
![image](https://github.com/Automatic-Survey-Generator/SurveyApp/assets/25122240/f15afcbb-e70f-4b5a-811e-755dcb6811ca)
Why are we removing this button?

Will it have this functionality?
![image](https://github.com/Automatic-Survey-Generator/SurveyApp/assets/25122240/ac767778-f7d0-4ebc-a003-fcea54231b5c)

Current:
![image](https://github.com/Automatic-Survey-Generator/SurveyApp/assets/25122240/03e56b29-2ef2-488f-a9cc-1b584a3692b6)

Reply:
Good catch, I think we should put that back